apps/web/package.json

```
@@ -8,6 +8,7 @@
     "build": "react-app-rewired build",
     "test": "react-app-rewired test",
     "precommit": "lint-staged",
+    "docker:build": "docker buildx build -f ./Dockerfile -t novu-web ./../..",
```

```suggestion
    "docker:build": "pnpm --silent --workspace-root pnpm-context -- apps/web/Dockerfile | docker buildx build --build-arg PACKAGE_PATH=apps/web - -t novu-web",
```
What do you think about to standardise the Docker building for all apps? I followed the same convention you used for API. Or is there a limitation for the Web app?

---

`PACKAGE_PATH` is used inside the Dockerfile of `api` which helps `pnpm` to install dependencies for the selected package and not all dependencies:

```
RUN --mount=type=cache,id=pnpm-store,target=/root/.pnpm-store\
 pnpm install --filter "novuhq" --filter "{${PACKAGE_PATH}}..."\
 --frozen-lockfile\
 --unsafe-perm\
 | grep -v "cross-device link not permitted\|Falling back to copying packages from store"

```